Vagrantfile

```diff
@@ -395,6 +421,32 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
         d.create_args = docker_args
         d.cmd = ["/bin/sh", "/opt/ce-vm-start.sh", "#{service_conf['docker_vagrant_user_uid']}", "#{service_conf['docker_vagrant_group_gid']}"]
       end
+      
+      ####### Triggers.
+      # We can not run more than one command under the same event.
+      # So we need to define the same event twice here (two .before : up)
+      #######
+
+      if(host_platform == "mac_os")
+        container.trigger.before :up do |trigger|
+          trigger.name = "ensure_lo_alias"
+          trigger.run = {inline: "sudo ifconfig lo0 alias #{net_ip}/32"}
+          trigger.warn = "Ensure loopback interface alias exists for #{net_ip}"
+        end
+      end
+
+      container.trigger.before :up do |trigger|
+        trigger.name = "ensure_docker_id"
+        ensure_docker_id(service, name)
+      end
+
+      container.trigger.after :provision do |trigger|
```

Comment:
We probably don't want to use a trigger here, because:
1: we don't need to run a command on the host
2: we need to copy those files before the last provisioner (that restart services)